hummingbot/core/gateway/__init__.py

```diff
@@ -94,31 +94,12 @@ def get_gateway_paths(client_config_map: "ClientConfigAdapter") -> GatewayPaths:
     if _default_paths is not None:
         return _default_paths
 
-    inside_docker: bool = is_inside_docker()
-
-    gateway_container_name: str = get_gateway_container_name(client_config_map)
     external_certs_path: Optional[Path] = os.getenv("CERTS_FOLDER") and Path(os.getenv("CERTS_FOLDER"))
     external_conf_path: Optional[Path] = os.getenv("GATEWAY_CONF_FOLDER") and Path(os.getenv("GATEWAY_CONF_FOLDER"))
     external_logs_path: Optional[Path] = os.getenv("GATEWAY_LOGS_FOLDER") and Path(os.getenv("GATEWAY_LOGS_FOLDER"))
-
-    if inside_docker and not (external_certs_path and external_conf_path and external_logs_path):
-        raise EnvironmentError("CERTS_FOLDER, GATEWAY_CONF_FOLDER and GATEWAY_LOGS_FOLDER must be defined when "
-                               "running as container.")
-
-    base_path: Path = (
-        Path.home().joinpath(".hummingbot-gateway")
-        if inside_docker
-        else Path.home().joinpath(f".hummingbot-gateway/{gateway_container_name}")
-    )
-    conf_path: Path = (
-        Path.home().joinpath("hummingbot-files")
-        if inside_docker
-        else Path.home().joinpath(f"hummingbot-files/{gateway_container_name}")
-
-    )
-    local_certs_path: Path = base_path.joinpath("certs")
-    local_conf_path: Path = conf_path.joinpath("gateway-conf")
-    local_logs_path: Path = conf_path.joinpath("gateway-logs")
+    local_certs_path: Path = client_config_map.certs
```

Comment:
Change param name to `certs_path` so that it's more descriptive and matches existing `log_file_path` param